redash/settings.py

```diff
@@ -122,6 +122,8 @@ def all_settings():
 
 JOB_EXPIRY_TIME = int(os.environ.get("REDASH_JOB_EXPIRY_TIME", 3600 * 6))
 COOKIE_SECRET = os.environ.get("REDASH_COOKIE_SECRET", "c292a0a3aa32397cdb050e233733900f")
+SESSION_COOKIE_SECURE = os.environ.get("REDASH_SESSION_COOKIE_SECURE", False)
```

Comment:
1. You should wrap it with `parse_boolean` like we do for the other boolean values (or it will be the string "True"/"true" instead of boolean value).

2. I think a nice default would be to set this to the value of `ENFORCE_HTTPS`. What do you think?